packages/components/src/styles/components/application-state.scss

```diff
@@ -7,43 +7,89 @@
 // APPLICATION-STATE COMPONENT
 //
 
+$hds-application-state-content-max-width: 480px;
+$hds-application-state-footer-child-gap: 8px;
 $hds-application-state-padding: 12px 0;
 
 .hds-application-state {
-  width: 19.5rem;
-  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+  max-width: 480px;
   margin: 0 auto; // this will center the component in the parent container
+
+  &.hds-application-state--align-center {
+    align-items: center;
+    text-align: center;
+
+    .hds-application-state__header,
+    .hds-application-state__body,
+    .hds-application-state__footer {
+      width: auto;
+    }
+  }
+}
+
+.hds-application-state__media {
+  margin-bottom: 32px;
+  overflow: visible;
+
+  // > * {
+  //   max-width: 100%;
+  // }
 }
 
 .hds-application-state__header {
   display: grid;
+  grid-template-rows: auto auto;
   grid-template-columns: min-content 1fr;
-  align-items: start;
-  color: var(--token-color-foreground-faint);
+  align-items: center;
+  width: 100%;
+  max-width: $hds-application-state-content-max-width;
+  margin: 0 auto;
+  color: var(--token-color-foreground-strong);
+}
+
+.hds-application-state__error-code {
+  grid-column: 1 / -1;
 }
 
 .hds-application-state__icon {
+  grid-row: 2;
   margin-right: 8px; // instead of column gap
   padding-top: 4px; // this seems to align the icon along with line 21
 }
 
-.hds-application-state__title,
-.hds-application-state__error-code {
-  grid-column-start: 2;
+.hds-application-state__title {
+  grid-row: 2;
+  font-weight: var(--token-typography-font-weight-semibold);
+
+  &:not(.hds-application-state__icon + &) {
+    grid-column: 1 / -1;
+  }
 }
 
 .hds-application-state__body {
-  padding: $hds-application-state-padding;
-  color: var(--token-color-foreground-faint);
+  width: 100%;
+  max-width: $hds-application-state-content-max-width;
+  margin: 12px auto;
```

Comment:
Design reference: 
<img width="362" alt="image" src="https://github.com/user-attachments/assets/ee479183-d67a-4c42-8251-1374bca8f49d">

```suggestion
  margin: 12px auto 24px;
```

Reply:
@majedelass what happens when there is a divider?

Reply:
The body has bottom-padding of 12px where the footer also has a top margin of 12px. Currently these together create a 24px gap. When the border is introduced, the gap is technically 25px (12px/1px/12px).